src/eslint/universal.js

```diff
@@ -0,0 +1,227 @@
+
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2022, // Allows for the parsing of modern ECMAScript features.
+    sourceType: 'module', // Allows for the use of imports.
+  },
+  env: {
+    node: true,
+    browser: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/all',
+    'plugin:import/recommended',
+    'plugin:import/typescript',
+    'plugin:unicorn/recommended',
+    'plugin:promise/recommended',
+    'plugin:lodash/recommended',
+  ],
+  plugins: [
+    '@typescript-eslint',
+    'deprecation',
+    'functional',
+    'prefer-arrow',
+    'unicorn',
+    'check-file',
+    'import',
+    'lodash',
+  ],
+  rules: {
+    /* Rule definitions and overrides for standard ESLint rules */
+    camelcase: 'error',
+    curly: ['error', 'multi-line', 'consistent'],
+    eqeqeq: 'error',
+    'no-await-in-loop': 'off', // Too restrictive, often false yields to more verbose code.
+    'no-console': ['error', { allow: ['info', 'groupCollapsed', 'groupEnd'] }],
+    'no-constant-condition': 'off', // Writing a "while(true)"" loop is often the most readable way to express the intent.
+    'no-fallthrough': 'off', // Does not work well with typescript exhaustive enums.
+    'no-inline-comments': 'off',
+    'no-lonely-if': 'error',
+    'no-nested-ternary': 'error',
+    'no-new-wrappers': 'error',
+    'no-return-await': 'off', // Superceded by @typescript-eslint/return-await.
+    'no-unexpected-multiline': 'off', // Conflicts with prettier.
+    'no-unused-expressions': 'off', // Superceded by @typescript-eslint/no-unused-expressions.
+    'object-shorthand': 'error',
+    'prefer-destructuring': [
+      'error',
+      {
+        array: false, // For arrays it is often confusing to use destructuring.
+        object: true,
+      },
+      {
+        enforceForRenamedProperties: false,
+      },
+    ],
+    'prefer-exponentiation-operator': 'error',
+    'prefer-named-capture-group': 'error',
+    'prefer-object-spread': 'error',
+    'prefer-spread': 'error',
+    'prefer-template': 'error',
+    radix: 'error',
+    // Sort keys does not have a fixer, but sorting lines can be trivially done by IDE (or some plugin). This rule has a
+    // nice configuration option "minKeys" which can specify how many keys should be present in an object before sorting
+    // is enforced. This is useful for small objects, where sorting is not necessary. Also, it allows creating groups
+    // (separated by newlines) which are sorted independently.
+    'sort-keys': ['error', 'asc', { caseSensitive: true, natural: true, minKeys: 10, allowLineSeparatedGroups: true }],
+    'spaced-comment': [
+      'error',
+      'always',
+      {
+        line: {
+          markers: ['/'],
+        },
+      },
+    ],
+
+    /* Rules to setup enforcement of arrow functions over regular functions */
+    'prefer-arrow/prefer-arrow-functions': [
```

Comment:
I'm not a fan of this "prefer arrow functions" rule because there are some good reasons to use function statements. Andre and I pointed out some reasons why we use function statements over here: https://github.com/api3dao/api3-dao-dashboard/pull/289#discussion_r873579942

Reply:
I also still strongly prefer function statements at the top level and use them in all my personal projects/scripts for the reasons I outlined in the linked post.

I think there was a discussion about this a while back (maybe it was a call?) where it was decided on to go with arrow functions everywhere (?). I can't remember exactly though.

Reply:
Yep. I rember the discussion as well. Will remove this rule.